### ISSUE TITLE
Non-descriptive error message

### ISSUE THREAD

**Issue by vshishkin-hot** [magento/magento2#36833]
### Preconditions and environment

- Magento version 2.4.4

### Steps to reproduce

1. Go to Sign up page
2. Navigate through the page using tab key with screen reader until reaching the required edit fields "First name, "Last name", etc.
3. Leave the required edit fields "First name, "Last name", etc as blank and submit the form.
4. Notice that the error messages "This is a required field", "This is a required field", etc are not descriptive.

### Expected result

Descriptive error messages are: "First name is required", "Last name is required", etc.

### Actual result

The error messages "This is a required field", "This is a required field", etc are not descriptive, the user is unable to identify what is wrong. 

Note: Instances of this issue were discovered. Please refer to the screenshots for details.

![image](https://user-images.githubusercontent.com/111375891/217821322-da0934d9-1193-4b1e-874e-7e9cab06269c.png)

### Triage and priority

- [ ] Severity: **S0** _- Affects critical data or functionality and leaves users without workaround._
- [ ] Severity: **S1** _- Affects critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S2** _- Affects non-critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S3** _- Affects non-critical data or functionality and does not force users to employ a workaround._
- [X] Severity: **S4** _- Affects aesthetics, professional look and feel, “quality” or “usability”._

**Comment by engcom-Dash** [comment]
@magento give me 2.4-develop instance

**Comment by engcom-Dash** [comment]
**Hi @vshishkin-hot ,**

Verified the issue in 2.4-develop instance and the issue is reproducible,Hence we are confirming the issue.

Magento Version 2.4-develop
PHP 8.1

**Steps:**
1.Install Magento instance
2.Go to front end
3.Click on Create an Account button
4.Navigate through the page using tab key with screen reader until reaching the required edit fields "First name, "Last name", etc.
5.Leave the required edit fields "First name, "Last name", etc as blank and submit the form.
**Screenshots**:
<img width="1512" alt="Screenshot 2023-02-10 at 1 50 26 PM" src="https://user-images.githubusercontent.com/60198592/218049077-908a2830-46a8-456b-8139-593e7615e6be.png">

As per the description we are getting actual result.

Let us know if we missing anything!!!

Thanks!!!!

**Comment by engcom-Dash** [comment]
Hi @vshishkin-hot ,

Thanks for ticket.
As per retesting and internal discussion with our team, since the behaviour is as per current core fields validation as implemented to the system hence this is as per expected behaviour.

Following the same  we are marking the ticket with Needs Update status.

Thanks

**Comment by vshishkin-hot** [comment]
Hi @engcom-Dash 
All fields has same error message "This is a required field".
The error messages "This is a required field", "This is a required field", etc are not descriptive, the user is unable to identify what is wrong.

Descriptive error messages are: "First name is required", "Last name is required", etc.

Please provide more descriptive text for the error so the user to understand what is wrong and how to fix it, making sure the error is clearly identified and associated with the corresponding input field.

Resources:
Descriptive error messages:
https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G85.html

**Comment by engcom-Lima** [comment]
@magento give me 2.4.3 instance

**Comment by engcom-Lima** [comment]
@magento give me 2.4.3 instance

**Comment by engcom-Lima** [comment]
Hi @vshishkin-hot ,

Thanks for your contribution and collaboration.

We have checked in our code base and referred this [comment](https://github.com/magento/magento2/issues/36833#issuecomment-1425646673) . We need to enhance our error messaging. Hence, marking this as a feature request.

Thanks
